### PR TITLE
cilium: bandwidth manager e2e tests

### PR DESCRIPTION
**Pull request by borkmann**
See commit description.

### CODE REVIEW

**Comment by borkmann** [comment]
test-focus K8sBandwidth*

**Comment by borkmann** [comment]
test-focus K8sBandwidth*

**Comment by borkmann** [comment]
test-focus K8sBandwidth*

**Comment by borkmann** [comment]
test-focus K8sBandwidth*

**Comment by borkmann** [comment]
test-me-please

File: test/k8sT/manifests/demo_bw.yaml
Comment (brb) [inline test/k8sT/manifests/demo_bw.yaml]: Could you tag the image and and add it to `cilium/packer-ci-build/provision/pull-images.sh`?

File: test/k8sT/Bandwidth.go
Comment (brb) [inline test/k8sT/Bandwidth.go]: Did you add the script to the cilium/netperf:latest image? If not, then it won't be available, and thus the exec at the line 117 will fail.

**Comment by borkmann** [comment]
test-focus K8sBandwidth*

**Comment by borkmann** [comment]
test-me-please

**Comment by borkmann** [comment]
test-focus K8sBandwidth*

File: test/k8sT/manifests/demo_bw.yaml
Comment (borkmann) [reply test/k8sT/manifests/demo_bw.yaml]: Added PR here https://github.com/cilium/packer-ci-build/pull/234

**Comment by borkmann** [comment]
test-focus K8sBandwidth*

**Comment by borkmann** [comment]
test-me-please

**Comment by borkmann** [comment]
test-focus K8sBandwidth*

**Comment by borkmann** [comment]
test-me-please

**Comment by borkmann** [comment]
test-focus K8sBandwidth*

**Comment by borkmann** [comment]
test-focus K8sBandwidth*

**Comment by borkmann** [comment]
test-me-please

**Review by brb — APPROVED** [review]

**Comment by borkmann** [comment]
test-focus K8sBandwidth*

**Comment by borkmann** [comment]
test-me-please

**Comment by borkmann** [comment]
test-me-please

**Comment by borkmann** [comment]
test-focus K8sBandwidth*

**Comment by borkmann** [comment]
test-focus K8sBandwidth*

**Comment by borkmann** [comment]
test-focus K8sBandwidth*

**Comment by borkmann** [comment]
test-me-please

**Comment by borkmann** [comment]
test-focus K8sBandwidth*

**Comment by borkmann** [comment]
test-me-please

**Comment by borkmann** [comment]
test-focus K8sBandwidth*

**Comment by borkmann** [comment]
test-me-please

File: test/k8sT/Bandwidth.go
Comment (christarazi) [inline test/k8sT/Bandwidth.go]: This should be `ExpectWithOffset` to be consistent with the rest of the expectations in this function.

File: test/k8sT/Bandwidth.go
Comment (christarazi) [inline test/k8sT/Bandwidth.go]: Should be `ExpectWithOffset` here too

**Review by christarazi — APPROVED** [review]
LGTM, minor changes required, but don't need a second review from me.

**Comment by borkmann** [comment]
test-me-please

**Comment by borkmann** [comment]
test-focus K8sBandwidth*

**Comment by borkmann** [comment]
test-me-please

**Comment by borkmann** [comment]
test-focus K8sBandwidth*

**Comment by borkmann** [comment]
test-me-please

**Comment by borkmann** [comment]
```
12:36:39  VBoxManage: error: Machine '4ac3ff21-058f-48c8-ae61-b5cebe2a3ca0' is not currently running
12:36:39  0%...10%...20%...30%...40%...
12:36:39  Progress state: VBOX_E_INVALID_OBJECT_STATE
12:36:39  VBoxManage: error: Machine delete failed
12:36:39  VBoxManage: error: Medium '/root/VirtualBox VMs/ubuntu-18.04.4-amd64-60_1598442716126_35661/ubuntu-18.04.4-amd64-60-disk001.vmdk' is locked for reading by another task
12:36:39  VBoxManage: error: Details: code VBOX_E_INVALID_OBJECT_STATE (0x80bb0007), component MediumWrap, interface IMedium
12:36:39  VBoxManage: error: Context: "RTEXITCODE handleUnregisterVM(HandlerArg*)" at line 162 of file VBoxManageMisc.cpp
12:36:40  0%...10%...20%...30%...40%...50%...60%...70%...80%...90%...100%
12:36:40  0%...10%...20%...30%...40%...50%...60%...70%...80%...90%...100%
12:36:40  0%...10%...20%...30%...40%...50%...60%...70%...80%...90%...100%
12:36:41  0%...10%...20%...30%...40%...50%...60%...70%...80%...90%...100%
12:36:41  0%...10%...20%...30%...40%...50%...60%...70%...80%...90%...100%
12:36:49  0%...10%...20%...30%...40%...50%...60%...70%...80%...90%...100%
12:36:49  Clean docker images from registry
```

**Comment by borkmann** [comment]
retest-net-next

**Comment by borkmann** [comment]
failure unrelated to test (https://jenkins.cilium.io/job/Cilium-PR-K8s-1.12-net-next/311/)

**Pull request merged**
